styles/_contribute.scss

```diff
@@ -0,0 +1,1158 @@
+        font-weight: 700;
+        line-height: 26px;
+        padding-bottom: 5px;
+        /* 130% */
+    }
+
+    .final-tr {
+        border: none !important;
+    }
+
+    #community-table h4 {
+        text-align: start;
+        margin: 10px 0;
+        font-family: Work Sans;
+        font-size: 32px;
+        font-style: normal;
+        font-weight: 700;
+        line-height: 38px;
+        /* 118.75% */
+    }
+
+    #community-table table {
+        width: 100%;
+        margin-top: 31px;
+        border-collapse: collapse;
+    }
+
+    #community-table tr {
+        padding: 10px 10px;
+        border-bottom: 1px solid #343437;
+    }
+
+    #community-table td {
+        overflow: hidden; // Hide the overflow
+        max-width: 34vw; // Your current max width
+        position: relative; // To position the pseudo-element
+        vertical-align: middle;
+    }
+
+    .scrolling-text {
+        display: inline-block;
+    }
+
+
+    #community-table img {
+        max-width: 52px;
+        height: 29px;
+    }
+
+    .td-img {
+        padding: 10px;
+        width: 69px;
+
+        .discord-icon {
+            content: url(../img/community/ic_Discord.png)
+        }
+
+        .twitter-icon {
+            content: url(../img/community/ic_Twitter.png);
+        }
+
+        .youtube-icon {
+            content: url(../img/community/ic_Youtube.png);
+        }
+
+        .xrpl-icon {
+            content: url(../img/community/ic_XRPL.png);
+        }
+
+        .github-icon {
+            content: url(../img/community/ic_GitHub.png);
+        }
+
+        .stackoverflow-icon {
+            content: url(../img/community/ic_StackOverflow.png);
+        }
+    }
+
+
+    .text-external-link {
+        display: inline-flex;
+        align-items: center;
+        margin-left: 10px;
+    }
+
+    .external-link-contribute {
+        display: inline-block;
+        vertical-align: middle;
+        padding-right: 41px;
+        height: 16px;
+        background: url(../img/icons/arrow-up-right.svg) no-repeat center center;
+        transition: transform 0.3s ease;
+        /* smooth transition effect */
+
+    }
+
+    .text-external-link:hover .external-link-contribute {
+        transform: translate(5px, -5px);
+        /* move 5px to the right and 5px up */
+    }
+
+    table td {
+        position: relative;
+        padding-right: 25px;
+        /* Give some space for the arrow */
+    }
+
+    table td .text-external-link {
+        position: absolute;
+        right: 5px;
+        top: 50%;
+        transform: translateY(-50%);
+    }
+
+    @media (max-width: 768px) {
+        #community-table img {
+            width: 96px;
+            height: 29px;
+        }
+
+        #community-table {
+            width: 100%;
+        }
+
+        .td-img {
+            min-width: 60px;
+            /* Remove fixed width for mobile view */
+        }
+    }
+
+    .funding-text {
+        color: var(--black-black-0, #FFF);
```

Comment:
What is going on with these color names? "black black zero"? We already have `$white`, and others defined in `_colors.scss`.